bigquery_etl/dryrun.py

```diff
@@ -154,6 +154,10 @@
         "sql/moz-fx-data-marketing-prod/acoustic/**/*.sql",
         recursive=True,
     ),
+    *glob.glob(
+        "sql/moz-fx-data-shared-prod/monitoring_derived/**/airflow_*.sql",
+        recursive=True,
+    ),  # noqa E501
```

Comment:
why is access denied to these tables?

Reply:
I believe that's because the SQL is trying to access Fivetran tables which the user used by the CI does not have access to.